rest_framework/response.py

```diff
@@ -67,18 +73,17 @@ def rendered_content(self):
             content_type = "{0}; charset={1}".format(media_type, charset)
         elif content_type is None:
             content_type = media_type
-        self['Content-Type'] = content_type
+        self["Content-Type"] = content_type
 
         ret = renderer.render(self.data, accepted_media_type, context)
         if isinstance(ret, six.text_type):
             assert charset, (
-                'renderer returned unicode, and did not specify '
-                'a charset value.'
+                "renderer returned unicode, and did not specify " "a charset value."
```

Comment:
These two strings can be one now. This is a known upstream bug https://github.com/ambv/black/issues/26.